source/03-components/_index.scss

```diff
@@ -34,3 +34,4 @@ $wysiwyg: false !default;
 @use 'view/view';
 @use 'view/views-view/views-view';
 @use 'view/views-view-unformatted/views-view-unformatted';
+@use 'call-to-action/call-to-action';
```

Comment:
@jbwagner Let's remove this and make the CSS file not a partial.  Then add it as a Drupal library.

Reply:
Fixed [6dd6fd9](https://github.com/forumone/gesso/pull/886/commits/6dd6fd9ac15e196e3d10c525df7a77e293b0471c)